lerna.json

```diff
@@ -1,7 +1,6 @@
 {
   "$schema": "node_modules/lerna/schemas/lerna-schema.json",
   "npmClient": "yarn",
-  "useWorkspaces": true,
```

Comment:
Context: workspaces are now used by default [since v7](https://github.com/lerna/lerna/blob/main/CHANGELOG.md#700-2023-06-08) and the config was removed.